lib/masamune/transform/postgres/snapshot_dimension.psql.erb

```diff
@@ -41,15 +41,15 @@ WITH ranges AS (
     <%- end -%>
     consolidated.start_at
   FROM (
-    SELECT DISTINCT ON (<%= target.window('start_at', 'id').join(', ') %>)
+    SELECT DISTINCT ON (<%= target.window('start_at').join(', ') %>)
       <%- target.insert_values(window: 'w').each do |value, last| -%>
       <%= value %><%= ',' %>
       <%- end -%>
       start_at AS start_at
     FROM
       windows
-    WINDOW w AS (PARTITION BY <%= target.window('window_id').join(', ') %> ORDER BY start_at <%= order %>)
-    ORDER BY <%= target.window("start_at #{order}", "id DESC", 'window_id').join(', ') %>
+    WINDOW w AS (PARTITION BY <%= target.window('window_id').join(', ') %> ORDER BY <%= ["start_at #{order}", "source_uuid #{order}"].join(', ') %>)
+    ORDER BY <%= target.window("start_at #{order}", "source_uuid DESC", 'window_id').join(', ') %>
```

Comment:
The `SELECT DISTINCT` is actually de-duplicating records that occur simultaneously, so `ORDER BY source_uuid DESC` is necessary to pick the latest source record when resolving conflicts